jd4/config.py

```diff
@@ -5,17 +5,22 @@
 from os import path
 from ruamel import yaml
 
+from jd4.log import logger
+
 _CONFIG_DIR = user_config_dir('jd4')
 _CONFIG_FILE = path.join(_CONFIG_DIR, 'config.yaml')
 _COOKIES_FILE = path.join(_CONFIG_DIR, 'cookies')
 
 def _load_config():
+    if not path.exists(_CONFIG_FILE):
+        logger.error('Config file %s not found.', _CONFIG_FILE)
+        exit(1)
     with open(_CONFIG_FILE) as file:
         return yaml.load(file, Loader=yaml.RoundTripLoader)
 
 config = _load_config()
 
-cookie_jar = CookieJar()
+cookie_jar = CookieJar(unsafe=True)
```

Comment:
unsafe (bool) – (optional) Whether to accept cookies from IPs.

Reference: http://aiohttp.readthedocs.io/en/stable/client_reference.html